bamboo/audit.sh

```diff
@@ -1,4 +1,16 @@
 #!/bin/bash
-npm install --no-audit
-npm run install-locks
-npm run audit
+commit_message_contains_skip_audit_flag=false
+commit_matches_version_tag=false
+
+if [[ $(git log --pretty='format:%Creset%s' -1) =~ '[skip-audit]' ]]; then
```

Comment:
I like this better as an if / then / else / fi, but not enough to block the merge.